snmp/datadog_checks/snmp/data/default_profiles/apc-netbotz.yaml

```diff
@@ -0,0 +1,355 @@
+extends:
+  - apc.yaml
```

Comment:
looking at the kentik netbotz profile it only extends `_base.yaml` however `apc.yaml` extends `_generic-tcp.yaml` and `_generic-udp.yaml` as well
(same for `apc_ups.yaml`)

Reply:
we can also remove the `_generic-tcp` and `_generic-udp` extends from the `apc.yaml` generic profile and add them to only `apc-ups` profile since it's the only profile extending them
note that the kentik profiles don't have the metadata that we have in `apc.yaml`, so not sure if this is applicable to new profiles or should be limited to `apc-ups` only ?

Reply:
Some Netbotz product uses an APC Management card that supports `_generic-tcp.yaml` and `_generic-udp.yaml`, that's why I extended `apc.yaml`
However not all of them does and some only support Netbotz mib, I'm not sure what is the best solution here, do you think I should remove `_generic-tcp.yaml` and `_generic-udp.yaml` ?

`apc_ups` is an old profile, it was made in 2020 (APC UPS do implement `TCP-MIB` and `UDP-MIB` from what I've seen)

Reply:
@AlexandreYang Okay, and make `apc.yaml` extends `_apc.yaml` ?

Reply:
- Unfortunately, I don't have a specific list. I will remove the `_generic-tcp` and `_generic-udp` extends and metadata from `apc-netbotz`, we can optionally add them later if needed
- For `apc-pdu`, PDUs implements the same MIB as UPS devices. The metadata we extract from sysDescr in `apc.yaml`  is not UPS specific, but more specific to the management card used in both UPS and PDU (if you look [here](https://github.com/DataDog/integrations-core/blob/8842af76ab30966bedabf9ca67500391a11576cb/snmp/datadog_checks/snmp/data/default_profiles/apc.yaml#L5) the example sysDescr given is for a pdu device)

Reply:
Would something like this work?

```
_apc.yaml             # vendor metadata only
_apc-metadata.yaml    # metadata from current apc.yaml (except vendor metadata)
apc.yaml
  extends:
    - _apc.yaml
    - _apc-metadata.yaml
apc-ups.yaml
  extends:
    - apc.yaml
apc-netboz.yaml    
  extends:
    - _apc.yaml
```

Reply:
we discussed this but wouldn't `_apc-metadata` be confusing because the naming means this applies to all `apc` profiles when it's not the case ?

Reply:
> we discussed this but wouldn't _apc-metadata be confusing because the naming means this applies to all apc profiles when it's not the case ?

I understand your point, but I think there are always exceptions (especially with acquisitions of companies).
I think it might fine if we consider it as `applies to all "normal" apc profiles except special cases like acquisitions`.

🤔